HyperPlay-Gaming/hyperplay-desktop-client
# [Feat] G7 Portal Webview

# Summary

closes https://github.com/HyperPlay-Gaming/product-management/issues/565

# Testing
Test case 1
1. Go to quests page
2. Click Earn Game7 Credits
3. See the summon portal webview

Test case 2
1. Go to quests page
2. Click View All Quests
3. See the HyperPlay quests page

# Screenshots
![image](https://github.com/user-attachments/assets/4d19ea48-b7ce-4794-8c40-afaffe5e807e)
![image](https://github.com/user-attachments/assets/35754d67-db61-40c4-a42c-ad898b70469f)
